src/app/common/components/search-additional-buttons/search-additional-buttons.interface.ts

```diff
@@ -13,7 +20,7 @@ export interface SearchAdditionalHeaderButtonInterface {
     iconSvgPath: string;
     svgClass: string;
     iconSvgPathClass: string;
-    counter: number;
+    counter?: number;
```

Comment:
I don't like this modeling, `counter`, `additionalOptions`. Not very far from "any" casts.
Please improve this model to have stricter typing rules.